src/test/java/seedu/bookmark/model/book/GoalTest.java

```diff
@@ -0,0 +1,55 @@
+package seedu.bookmark.model.book;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.bookmark.testutil.BookBuilder;
+
+public class GoalTest {
```

Comment:
Perhaps a test for the constructor with null and invalid values would make for more comprehensive testing